Libraries/LibGfx/CMakeLists.txt

```diff
@@ -118,8 +118,17 @@ find_package(LIBAVIF REQUIRED)
 find_package(WebP REQUIRED)
 pkg_check_modules(harfbuzz REQUIRED IMPORTED_TARGET harfbuzz)
 
-target_link_libraries(LibGfx PRIVATE PkgConfig::WOFF2 JPEG::JPEG PNG::PNG avif WebP::webp WebP::webpdecoder
+if (HAS_FONTCONFIG)
+    pkg_check_modules(fontconfig REQUIRED IMPORTED_TARGET fontconfig)
```

Comment:
If we HAS_FONTCONFIG, that means we already checked that fontconfig exists. We should be using the imported targets from the `Fontconfig` package, not from pkg_check_modules.